proto/stride/stakeibc/tx.proto

```diff
@@ -50,6 +52,17 @@ message MsgLSMLiquidStake {
 }
 message MsgLSMLiquidStakeResponse { bool transaction_complete = 1; }
 
+message MsgCommunityPoolLiquidStake {
```

Comment:
we discussed this on slack - I think you can go ahead and remove this tx cause we can run the test without it!

Reply:
So there are two purposes.  One is to run a test, the second is if any other denom besides DYDX or stDYDX makes it into the deposit ICA, this tx would allow them to eventually return those funds to the community pool.  Otherwise, any 3rd party denom which is in the deposit ICA would be stuck there.  

So if somehow there is a 3rd party token like ATOM in the pool and it mistakenly gets put into the deposit ICA, then running this tx would transfer (without stake or unstake autopilot action) it to the holding module address on Stride.  Then on the next epoch, everything regardless of denom in the holding address will be transferred to the return ICA and eventually be used to fund-community-pool.

I do kind of agree that the low likelihood but real race condition between the epochly checks and the tx that you pointed out might be justification to just remove it completely.  Maybe we just say it is their mistake if any tokens other than DYDX or stDYDX ever show up in the deposit ICA and don't try to address it in this iteration.

Reply:
yeah from our discussion the other day, I think we're all aligned that if someone accidentally places it there, it's kind of on them. There are a few attack vectors with this tx (see slack). 

But even attack vectors aside, txs in general have a lot of boilerplate so I don't think we should bear the burden of having to add more complexity to our codebase to support this case when we could always refund through an upgrade if needed! (and realistically no one is going to accidentally send funds here haha)